dev/AppNotifications/ShellLocalization.cpp

```diff
@@ -258,10 +260,7 @@ HRESULT Microsoft::Windows::AppNotifications::ShellLocalization::DeleteIconFromC
     std::path iconFilePath{ RetrieveLocalFolderPath() / (notificationAppId + c_pngExtension) };
 
     // If DeleteFile returned FALSE, then deletion failed and we should return the corresponding error code.
-    if (DeleteFile(iconFilePath.c_str()) == FALSE)
-    {
-        THROW_HR(HRESULT_FROM_WIN32(GetLastError()));
-    }
+    winrt::check_bool(DeleteFile(iconFilePath.c_str()) != FALSE);
```

Comment:
NIT: `DeleteFileW` is preferable to `DeleteFile` (ensures no ambiguity to the reader)

Is `winrt::check_bool()` equivalent `wil::RETURN_IF_WIN32_BOOL_FALSE()`?

NOTE: It's weird bordering on perverse to explicitly check for == or != for values of `TRUE` or `FALSE`